Makefile

```diff
@@ -10,6 +10,9 @@ ADMIN_CTX ?= ""
 REGISTRY_ENABLED ?= true
 DEVWORKSPACE_API_VERSION ?= master
 CERT_IMG ?= quay.io/che-incubator/che-workspace-controller-cert-gen:latest
+TERMINAL_MANIFEST_VERSION ?= master
+BUNDLE_IMG ?= quay.io/che-incubator/che-workspace-bundle:latest
```

Comment:
Are we going to build those images with CI?
If no - then probably it's better to leave them empty and require before running OLM related tasks.

Reply:
Hmm, if I understand how everything fits together with OLM we need to re-build the bundle img for each release and then add them to the new index image but we don't need nightly builds